zerver/webhooks/raygun/doc.md

```diff
@@ -0,0 +1,30 @@
+Raygun is an error and performance monitoring for web and mobile apps. Follow
+these steps to integrate your Raygun application to Zulip:
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+{!append-topic.md!}
+
+Copy the URL and do the following steps:
+
+1. Go to your [Raygun dashboard](https://app.raygun.com/dashboard/) and select
+   your application (top left):
+   ![](/static/images/integrations/raygun/001.png)
+
+2. Click on Integrations, located at the left sidebar:
+   ![](/static/images/integrations/raygun/002.png)
```

Comment:
This doesn't look good when rendered, that is the text and the image are shown on the same line, maybe add a line break here?

![image](https://user-images.githubusercontent.com/1736001/34650799-b08490b6-f3ec-11e7-8cfc-ffe873a493bc.png)

Reply:
ah, that was intentional. Will add line break though for the next push